### ISSUE TITLE
g_loss is quite large while d_loss is almost zero

### ISSUE THREAD

**Issue by airsyuan** [xuan-li/zi2zi-pytorch#2]
Hi, friend. I am new to style-transfer and very interesting in this project. After training for several epochs, only to find  g_loss is quite large while d_loss is almost zero.  I am not sure if the loss function leads to this problem.  Could you help me to solve the problem? Thanks a lot.

**Comment by xuan-li** [comment]
Sorry, I am currently interning at Adobe. I have no time to fully test this model, so there may be some bugs in my code. I will continue on this model when I come back to university :)

**Comment by airsyuan** [comment]
Many thanks for your reply.  If this project has been test successfully, maybe I need to check if my dataset is hard to fit.

**Comment by luotom** [comment]
> Many thanks for your reply. If this project has been test successfully, maybe I need to check if my dataset is hard to fit.
Hi, friend. If you solved the problem, I also met the problem you described when g_loss is quite large while d_loss is almost zero. And the training results were not satisfactory like the flow.
![捕获](https://user-images.githubusercontent.com/43427583/99133403-cccc9500-2654-11eb-9e6d-59ebabc1d45a.PNG)
